lib/gui/modules/progress-status.js

```diff
@@ -41,7 +41,7 @@ exports.fromFlashState = (state) => {
   const shouldValidate = settings.get('validateWriteOnSuccess')
   const shouldUnmount = settings.get('unmountOnSuccess')
 
-  if (state.percentage === utils.PERCENTAGE_MINIMUM && !state.speed) {
+  if (state.percentage === utils.PERCENTAGE_MINIMUM && state.speed === '0 B/s') {
```

Comment:
I think we should leave `state.speed` be, and keep it a number, but add `state.humanSpeed` or something that contains the speed as a human readable string, because we might want to use the `state.speed` in other places as a number.